routers/web/repo/view.go

```diff
@@ -758,7 +759,7 @@ func checkHomeCodeViewable(ctx *context.Context) {
 		}
 	}
 
-	ctx.NotFound("Home", fmt.Errorf(ctx.Tr("units.error.no_unit_allowed_repo")))
+	ctx.NotFound("Home", errors.New("units.error.no_unit_allowed_repo"))
```

Comment:
No traslation now?

Reply:
The second argument is for "log error". It doesn't make sense to output translated message into log.

It is the old code's problem.

Reply:
Reverted to old code by 32f0cd31729c4bb4da242c3fc20c6a1258031029